validator/validation/validation.py

```diff
@@ -352,13 +366,26 @@ def run_validation(validation_id):
                                 validation_aborted = True
                                 __logger.debug('Validation got cancelled, dropping task {}: {}'.format(async_result.id, te))
 
-                if validation_aborted:
+                # in case there where no points with overlapping data within
+                # the validation period, results is an empty dictionary, and we
+                # count this job as error
+                if validation_aborted or not results:
                     validation_run.error_points += num_gpis_from_job(job_table[async_result.id])
                 else:
+                    results = pytesmo_to_qa4sm_results(results)
+
+                    # only count points where there was enough data as ok points
```

Comment:
What is the reason for this part commented out here and in line 384?

Reply:
This was actually for a future PR I planned, but I think I'll just move it to another branch.